db/models/task.js

```diff
@@ -5,10 +5,11 @@ module.exports = {
         task = orm.define('Tasks', {
           id         : {type: Sequelize.INTEGER, primaryKey: true, autoIncrement: true},
           content 	 : {type: Sequelize.TEXT, allowNull: false},
-          parent     : Sequelize.INTEGER,
-          isCompleted: Sequelize.BOOLEAN,
+          parentId   : {type: Sequelize.INTEGER, defaultValue: 0},
```

Comment:
Is this parentId or parent? as one on the post-install scripts  (db/seed/initial_tasks.js) complains about "table Task has no  column named  parentId".